package.json

```diff
@@ -19,24 +19,28 @@
     "gen:rates": "openapi-generator-cli generate -i apps/rates-service/src/api/swagger.json -g typescript-axios -o apis/rates/ts -c apis/rates/client-ts-spec.json",
     "gen:broker": "openapi-generator-cli generate -i apps/broker-service/src/api/swagger.json -g typescript-axios -o apis/broker/ts -c apis/broker/client-ts-spec.json",
     "gen:nft": "openapi-generator-cli generate -i apps/nft-service/src/api/swagger.json -g typescript-axios -o apis/nft/ts -c apis/nft/client-ts-spec.json",
-    "genapis": "run-p gen:*",
-    "build:projects": "lerna run build",
-    "build:rates": "cd apis/rates/ts && yarn build",
-    "build:broker": "cd apis/broker/ts && yarn build",
-    "build:nft": "cd apis/nft/ts && yarn build",
-    "buildapis": "run-p build:*",
-    "build": "cross-env CONFIG_NAME=prod run-s clean build:projects genapis buildapis",
+    "genapis": "lerna run gen:swagger && run-p gen:*",
+    "build:libs": "lerna run build:lib",
+    "build:apps": "lerna run build:app --stream",
+    "build:apis": "run-p build:api:*",
+    "build:api:rates": "cd apis/rates/ts && yarn build",
+    "build:api:broker": "cd apis/broker/ts && yarn build",
+    "build:api:nft": "cd apis/nft/ts && yarn build",
+    "build:all": "run-s clean build:libs genapis build:apis build:apps",
+    "build": "cross-env CONFIG_NAME=prod yarn build:all",
     "dev:live:firestore": "yarn firebase emulators:start --only firestore --project broker-cad",
     "dev:live:ceramic": "yarn ceramic daemon",
     "dev:live:blockchain": "cd libs/contract && yarn truffle develop",
     "dev:live:projects": "yarn wait-port 8377 && yarn lerna run dev:live --stream --no-bail",
     "dev:live:bootstrap": "yarn wait-port 7007 && yarn idx bootstrap && node ./tools/bootstrapIdx.js",
     "dev:live": "cross-env CONFIG_NAME=devlive run-p dev:live:*",
-    "deploy:version": "lerna version 0.2.1 --allow-branch publish/* --no-push",
     "deploy:prodtest": "cross-env CONFIG_NAME=prodtest yarn _deploy",
     "deploy:prod": "cross-env CONFIG_NAME=prod yarn _deploy",
-    "pre_deploy": "yarn build",
-    "_deploy": "lerna run deploy",
+    "pre_deploy": "yarn build:all",
+    "_deploy": "run-s _deploy:version _deploy:lib _deploy:app",
+    "_deploy:version": "lerna version --allow-branch */* --no-push",
+    "_deploy:lib": "lerna run deploy:lib",
+    "_deploy:app": "lerna run deploy:app",
```

Comment:
Our build steps have become much too specific.  The next PR is going to focus on removing some (hopefully unnecessary) steps in this process